kerykeion/settings/translation_strings.py

```
@@ -907,6 +939,14 @@
         "heliocentric": "日心",
         "topocentric": "站心",
         "true_geocentric": "真地心",
+        "selenocentric": "月心",
+        "mercurycentric": "水星心",
+        "venuscentric": "金星心",
+        "marscentric": "火星心",
+        "jupitercentric": "木星心",
+        "saturncentric": "土星心",
+        "barycentric": "质心",
+        "chart_contents": "{points} 个点，{aspects} 个相位",
```

Ruff (0.16.2)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)